src/test/shared/filesystemUtilities.test.ts

```diff
@@ -101,33 +101,17 @@ describe('filesystemUtilities', () => {
         })
     })
 
-    describe('isContainedWithinDirectory', () => {
+    it('isInDirectory()', () => {
         const basePath = path.join('this', 'is', 'the', 'way')
         const extendedPath = path.join(basePath, 'forward')
         const filename = 'yadayadayada.log'
 
-        it('returns true for the same dir', () => {
-            assert.ok(isContainedWithinDirectory(basePath, basePath))
-        })
-
-        it('returns true for a subdir', () => {
-            assert.ok(isContainedWithinDirectory(basePath, extendedPath))
-        })
-
-        it('returns true for a file in the same dir', () => {
-            assert.ok(isContainedWithinDirectory(basePath, path.join(basePath, filename)))
-        })
-
-        it('returns true for a file in a subdir', () => {
-            assert.ok(isContainedWithinDirectory(basePath, path.join(extendedPath, filename)))
-        })
-
-        it('returns false for a completely different dir', () => {
-            assert.ok(!isContainedWithinDirectory(basePath, path.join('what', 'are', 'you', 'looking', 'at')))
-        })
-
-        it('returns false for a similarly named dir', () => {
-            assert.ok(!isContainedWithinDirectory(basePath, `${basePath}point`))
-        })
+        assert.ok(isInDirectory(basePath, basePath))
+        assert.ok(isInDirectory(basePath, extendedPath))
+        assert.ok(isInDirectory(basePath, path.join(basePath, filename)))
+        assert.ok(isInDirectory(basePath, path.join(extendedPath, filename)))
+        assert.ok(!isInDirectory(basePath, path.join('what', 'are', 'you', 'looking', 'at')))
+        assert.ok(!isInDirectory(basePath, `${basePath}point`))
+        assert.ok(isInDirectory('/foo/bar/baz/', '/foo/bar/baz/a.txt'))
```

Comment:
This test fails before the fix.

Reply:
My comment earlier (https://github.com/aws/aws-toolkit-vscode/pull/1036/files/e7230ff600e3a1e914f1a17c7bdc6d328d6ad89f..b4f8d1d37bd1cd5b9c867290b0257de57f26ae5f#r409049401) failed to render correctly.  Should there be a test that uses the explicit `\` slash in pathing as well?

Reply:
done